pkg/pod/resource_request.go

```diff
@@ -41,7 +41,7 @@ func resolveResourceRequests(containers []corev1.Container) []corev1.Container {
 		}
 	}
 
-	// Set resource requests for all steps but the theh last container to
+	// Set resource requests for all steps but the the last container to
```

Comment:
Ah, correct spelling but we actually don't need the second `the` - this should become `but the last container`

Reply:
I think I'm dazzled and need a rest... I force push to my branch for fixing it.